datamade/just-spaces
# Deleting/deactivating surveys

## Overview

Closes #84. In design review with Derek we decided to treat deactivation as a soft delete, and refer to it as such on the site. This PR adds buttons to delete (read: deactivate) both published and unpublished surveys. It also creates a `deactivate.html` partial to cut down on repetitive code on the templates for deleting locations, studies, and surveys.

## Testing Instructions

* Run migrations
* Create a new survey. Make sure you can see it in the "Edit Surveys" view
* Delete that survey. Make sure it disappears
* Create a new published survey or pick an existing one. 
* Delete it. Make sure that deactivated survey no longer shows up in the "Run survey" list
